airbyte-integrations/connectors/destination-postgres-strict-encrypt/src/main/java/io/airbyte/integrations/destination/postgres/PostgresDestinationStrictEncrypt.java

```diff
@@ -25,6 +26,11 @@ public PostgresDestinationStrictEncrypt() {
   public ConnectorSpecification modifySpec(final ConnectorSpecification originalSpec) {
     final ConnectorSpecification spec = Jsons.clone(originalSpec);
     ((ObjectNode) spec.getConnectionSpecification().get("properties")).remove("ssl");
+    ArrayNode modifiedSslModes = spec.getConnectionSpecification().get("properties").get("ssl_mode").get("oneOf").deepCopy();
```

Comment:
we should also remove "disable" and "allow" modes from the strict encrypt version

Reply:
as I understood from the documentation, it is more correct that the `common name (CN)` should be different for root certificates and server certificates. For cases like ours, it is recommended to use the IP address for the root certificate, and the localhost for the server certificate. These changes will not break the source or destination tests.